src/app.js

```diff
@@ -31,7 +31,7 @@ async function run() {
 		check();
 	} catch (error) {
 		logError(error);
-		check();
+		process.exit(5);
```

Comment:
Is this intentionally `5`? We don't need to be too specific with OS-specific exit codes, so `1` would be sufficient.